backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/InlineClassPropertyAccessorsLowering.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package org.jetbrains.kotlin.backend.konan.lower
+
+import org.jetbrains.kotlin.backend.common.*
+import org.jetbrains.kotlin.backend.konan.*
+import org.jetbrains.kotlin.ir.declarations.*
+import org.jetbrains.kotlin.ir.expressions.*
+import org.jetbrains.kotlin.ir.visitors.*
+
+/**
+ * Inline accessors of inline class properties.
+ * TODO: this lowering is needed because common lowering PropertyAccessorInlineLowering
```

Comment:
Also PropertyAccessorInlineLowering produces access to backing field, but accessing backing fields of inline classes should be lowered by this moment.

Reply:
Fix comment?

Reply:
Yes, if we don't manage to meld this lowering into InlineClassTransformer.